src/main/java/Duke.java

```diff
@@ -1,10 +1,145 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String line = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    private static final Scanner input = new Scanner(System.in);
+
+    private static final List<Task> tasks = new ArrayList<Task>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        welcomeMessage();
+
+        String userInput;
+         do {
+            System.out.println(line);
+            userInput = input.nextLine().trim();
+            System.out.println(line);
+        } while(userCommand(userInput));
+    }
+
+    /**
+     * Prints welcome message for the bot.
+     */
+    private static void welcomeMessage() {
+        System.out.println(line);
+        System.out.println("                Hewwo! I'm UwU Bot\n"
+                          +"              Hwere to hwelp you out");
+        System.out.println(line);
+        System.out.println("What can I dwo fwor ywou?");
+    }
+
+    /**
+     * Chooses the appropriate command base on user's input.
+     * Returns false if user enters "bye" otherwise returns true.
+     *
+     * @param userInput user's input.
+     * @return boolean value.
+     */
+    private static boolean userCommand(String userInput) {
```

Comment:
I think it's the same for the other methods below - maybe you can consider naming them as getUserCommand() or something that better describes what the method does?

Reply:
Will look through all my methods, probably more with this issue. Thanks!